frontend/src/lib/components/PropertyFilters/PropertyValue.js

```diff
@@ -1,66 +1,65 @@
 import React, { Component } from 'react'
 import api from '../../api'
-import AsyncCreatableSelect from 'react-select/async-creatable/dist/react-select.esm'
-import { selectStyle, debounce } from '../../utils'
+import { Select } from 'antd'
+import { debounce } from '../../utils'
 import PropTypes from 'prop-types'
 
 export class PropertyValue extends Component {
     constructor(props) {
         super(props)
-        this.state = { input: props.value }
+        this.state = {
+            input: '',
+            optionsCache: [],
+            options: props.operator === 'is_set' ? ['true', 'false'] : [],
+        }
+
         this.loadPropertyValues = debounce(this.loadPropertyValues.bind(this), 250)
-        this.ref = React.createRef()
+        if (this.props.operator !== 'is_set') {
```

Comment:
This is only checking during initialization, but the operator can change while the component is still mounted and the options won't be refreshed